openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala

```diff
@@ -338,6 +341,7 @@ case class MediaType(schema: Option[Schema])
 sealed trait Schema {
   def description: Option[String]
```

Comment:
BTW, did you find any place where we used to set a `description`? I couldn’t find one.

Reply:
Yeah, I think it used to be set in `withDefinedDescription`, which was used for setting field property docs and for enumeration descriptions.